### PR TITLE
Initial implementation of swagger documentation.

### PR DESCRIPTION
**Pull request by Kang2001**

### CODE REVIEW

**Review by davidfrisch — CHANGES_REQUESTED** [review]
As talked in WhatsApp, the swagger has to be included in the main flask server.

**Review by davidfrisch — CHANGES_REQUESTED** [review]
Please remove the .idea from the commit

**Review by davidfrisch — CHANGES_REQUESTED** [review]
Is it possible to reduce this file and move the titles and functino that combines into another file ? You should create a folder docs and put the json files in it.

**Comment by Kang2001** [comment]
> Is it possible to reduce this file and move the titles and functino that combines into another file ? You should move it in the swagger docs

yes, I will change it.

**Comment by Kang2001** [comment]
> Thanks Weihao, looks better and better.
> 
> However, the endpoints are not correct. 
> 
> Please find attached the latest endpoints.
> 
> [WakeUpServer.postman_collection.json](https://github.com/SSE-GROUP5/wakeup-system/files/14129484/WakeUpServer.postman_collection.json)
> 
> 

Thank you David, I will check it again.

File: swagger-doc/wakeup-server/Json/switch_api.json [hunk 1]
```diff
@@ -0,0 +1,96 @@
+{
+  "swagger": "2.0",
+  "info": {
+    "version": "1.0.0",
+    "title": "Switch API",
+    "description": "API for controlling a Home Assistant Switch"
+  },
+  "host": "localhost:5000",
```
[the remark this answers is not in the document]
Comment (Kang2001) [reply swagger-doc/wakeup-server/Json/switch_api.json:8]: Yes, I will change it.

**Review by davidfrisch — APPROVED** [review]

**Pull request merged**